src/fides/api/ops/api/v1/endpoints/privacy_notice_endpoints.py

```diff
@@ -48,22 +70,60 @@ def get_privacy_notice_list(
     systems_applicable: Optional[bool] = False,
 ) -> AbstractPage[PrivacyNotice]:
     """
-    Return a paginated list of PrivacyNotice records in this system.
+    Return a paginated list of `PrivacyNotice` records in this system.
     Includes some query params to help filter the list if needed
     """
     logger.info("Finding all PrivacyNotices with pagination params '{}'", params)
-    notice_query = db.query(PrivacyNotice)
-    if not show_disabled:
-        notice_query = notice_query.filter(PrivacyNotice.disabled.is_(False))
-    if region is not None:
-        notice_query = notice_query.filter(PrivacyNotice.regions.contains([region]))
-    if systems_applicable:
-        data_uses = System.get_system_data_uses(db)
-        notice_query = notice_query.filter(PrivacyNotice.data_uses.overlap(data_uses))  # type: ignore
+    notice_query = generate_notice_query(
+        db=db,
+        show_disabled=show_disabled,
+        systems_applicable=systems_applicable,
+        region=region,
+    )
     privacy_notices = notice_query.order_by(PrivacyNotice.created_at.desc())
     return paginate(privacy_notices, params=params)
 
 
+@router.get(
+    urls.PRIVACY_NOTICE_BY_DATA_USE,
```

Comment:
Can we add this to the postman collection while we're here?

Reply:
yes! good call, thanks for the reminder